website_sale_product_minimal_price/__manifest__.py

```diff
@@ -12,7 +12,7 @@
     "license": "AGPL-3",
     "application": False,
     "installable": True,
-    "depends": ["website_sale"],
+    "depends": ["website_sale_stock"],
```

Comment:
Why changing this dependency?

Reply:
Because the module that adds the method `VariantMixin._onChangeCombinationStock` is this other

Reply:
And can we avoid that one and use one in pure `website_sale` module? It's not logical to require stock for this. What about if you sell services?

Reply:
Ok I will try to use the website_sale module